go/chat/utils/utils.go

```diff
@@ -360,20 +360,22 @@ func IsVisibleChatMessageType(messageType chat1.MessageType) bool {
 	return false
 }
 
-func IsNotifiableChatMessageType(messageType chat1.MessageType, atMentions []gregor1.UID, chanMention chat1.ChannelMention) bool {
+func IsNotifiableChatMessageType(messageType chat1.MessageType, atMentions []gregor1.UID,
+	chanMention chat1.ChannelMention) bool {
 	if IsVisibleChatMessageType(messageType) {
 		return true
 	}
-
-	if messageType != chat1.MessageType_EDIT {
-		return false
-	}
-
-	// an edit with atMention or channel mention should generate notifications
-	if len(atMentions) > 0 || chanMention != chat1.ChannelMention_NONE {
+	switch messageType {
+	case chat1.MessageType_EDIT:
+		// an edit with atMention or channel mention should generate notifications
+		if len(atMentions) > 0 || chanMention != chat1.ChannelMention_NONE {
+			return true
+		}
+	case chat1.MessageType_REACTION:
+		// effect of this is all reactions will notify if they are sent to a person that
+		// is notified for any messages in the conversation
```

Comment:
some changes are missing here but are in the server pr: https://github.com/keybase/keybase/pull/3044/files#diff-0c41f3030b7365bba4b704fc82318c25R1358

Reply:
Those are changes that I made in some old client PR that never got vendored into the server, but are now coming for the ride.

Reply:
ah makes sense just wanted to double check